OrbitQt/Error.h

```diff
@@ -14,7 +14,9 @@ enum class Error {
   kCouldNotUploadPackage,
   kCouldNotUploadSignature,
   kCouldNotInstallPackage,
-  kCouldNotStartTunnel
+  kCouldNotStartTunnel,
+  kUserCanceledServiceDeployment,
```

Comment:
Cancelled?

Reply:
I specifically looked that up 😅 
According to grammarly.com: Americans prefer canceled, Britons cancelled. 
(Also learned that its called Britons today)

Therefore, I am going to leave it as is